src/docs/asciidoc/data-access.adoc

```diff
@@ -2777,26 +2774,25 @@ The following query finds and populates a number of domain objects:
 
 
 If the last two snippets of code actually existed in the same application, it would make
-sense to remove the duplication present in the two `RowMapper` anonymous inner classes
-and extract them out into a single class (typically a `static` nested class) that could
-then be referenced by DAO methods as needed. For example, it may be better to write the
+sense to remove the duplication present in the two `RowMapper` lambda expressions and
+extract them out into a single expression or class (typically a `static` nested class) that
+could then be referenced by DAO methods as needed. For example, it may be better to write the
 preceding code snippet as follows:
 
 [source,java,indent=0,subs="verbatim,quotes",role="primary"]
 .Java
 ----
 	public List<Actor> findAllActors() {
-		return this.jdbcTemplate.query( "select first_name, last_name from t_actor", new ActorMapper());
+		return this.jdbcTemplate.query( "select first_name, last_name from t_actor", getActorMapper());
 	}
 
-	private static final class ActorMapper implements RowMapper<Actor> {
-
-		public Actor mapRow(ResultSet rs, int rowNum) throws SQLException {
+	private RowMapper<Actor> getActorMapper() {
```

Comment:
The `RowMapper` implementation is stateless. Therefore, please store a single reference to it in a `final` field and remove the `getActorMapper()` method.

Reply:
Ok, stored as instance field `actorRowMapper`. Let know static field `ACTOR_ROW_MAPPER` is preferred